var/spack/repos/builtin/packages/fzf/package.py

```diff
@@ -37,7 +35,9 @@ class Fzf(MakefilePackage):
 
     @classmethod
     def determine_version(cls, exe):
-        return Executable(exe)('--version', output=str, error=str).rstrip()
+        candidate = Executable(exe)('--version', output=str, error=str)
+        match = re.match(r'(^[\d.]*)', candidate)
```

Comment:
I'm not sure if/when it would come up, but shouldn't this be:

```suggestion
        match = re.match(r'(^[\d.]+)', candidate)
```

So the match has to include at least one character and we don't get the empty string returned instead of `None` on a non-match?